clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleLoadBalancerCachingAgent.groovy

```diff
@@ -234,7 +234,7 @@ class GoogleLoadBalancerCachingAgent extends AbstractGoogleCachingAgent implemen
   class HttpHealthCheckCallback<HttpHealthCheck> extends JsonBatchCallback<HttpHealthCheck> implements FailureLogger {
 
     GoogleLoadBalancer googleLoadBalancer
-    TargetPool targetPool
+    def targetPool
```

Comment:
We were encountering what looked like this: https://jira.atlassian.com/plugins/servlet/mobile#issue/CLOV-1779/comment/778908

Cache population was failing with NPEs resulting in no populated load balancers.